src/toolchain.rs

```diff
@@ -119,4 +175,35 @@ impl Toolchain {
 
         Ok(download_cfg)
     }
+
+    pub fn remove_component(&self, component: &str) -> Result<()> {
+        if self.has_component(component) {
+            info!("Removing '{}' from toolchain '{}'", component, self.name);
+            let component_path = self.path.join(component);
+            remove_file(component_path)?;
+            // If component to remove is 'forc', silently remove forc plugins
+            if component == component::FORC {
+                for component in [
+                    component::FORC_FMT,
+                    component::FORC_LSP,
+                    component::FORC_EXPLORE,
+                ] {
```

Comment:
Maybe we should consider moving this list to a `const` array, e.g. `component::FORC_PLUGINS`. It will be important to have an easy-to-find, central source of truth for this list as we add new plugins.

Reply:
Just remembered there was a `SUPPORTED_PLUGINS` that i created a while back that I can use